packages/gatsby/src/redux/reducers/layouts.js

```diff
@@ -0,0 +1,36 @@
+const _ = require(`lodash`)
+const normalize = require(`normalize-path`)
+
+module.exports = (state = [], action) => {
+  switch (action.type) {
+    case `DELETE_CACHE`:
+      return []
+    case `CREATE_LAYOUT`:
+      action.payload.component = normalize(action.payload.component)
+      if (!action.plugin && !action.plugin.name) {
+        console.log(``)
```

Comment:
we really should standardize this output. maybe attach the json to the error instance instead of logging out?